src/root/views/ThreeW/project-details.js

```diff
@@ -144,7 +144,10 @@ class ProjectDetails extends React.PureComponent {
               />
               <TextOutput
                 label={strings.threeWDisasterType}
-                value={dtype ? disasterTypes[dtype] : undefined}
+                value={dtype
+                  ? this.props.disasterTypesSelect.filter(dt => dt.value === dtype).label
```

Comment:
@frozenhelium @batpad Couldn't find out how to test this locally, I hope it works...

Reply:
@frozenhelium would you be able to help test this?

Reply:
@batpad @frozenhelium I had to reproduce the same usage anyway and found out I need to use `.find()` here instead of `.filter()` and check if `.find()` is null-proof or not. Will be fixed soon.

Reply:
@GregoryHorvath there might be new field `dtype_display`, we can directly use that instead (if there is one)

Reply:
It's actually `dtype_details?.name`

Reply:
@frozenhelium thanks! Was able to use this at a couple of places.

Reply:
44c22ff